generated/new-lspconfig-servers.json

```diff
@@ -0,0 +1,373 @@
+  "circom-lsp": "[Circom Language Server](https://github.com/rubydusa/circom-lsp)\n`circom-lsp`, the language server for the Circom language.",
+  "clangd": "https://clangd.llvm.org/installation.html\n- **NOTE:** Clang >= 11 is recommended! See [#23](https://github.com/neovim/nvim-lspconfig/issues/23).\n- If `compile_commands.json` lives in a build directory, you should\n  symlink it to the root of your source tree.\n  ```\n  ln -s /path/to/myproject/build/compile_commands.json /path/to/myproject/\n  ```\n- clangd relies on a [JSON compilation database](https://clang.llvm.org/docs/JSONCompilationDatabase.html)\n  specified as compile_commands.json, see https://clangd.llvm.org/installation#compile_commandsjson",
+  "clarinet": "https://github.com/hirosystems/clarinet\nClarinet is the fastest way to build, test, and deploy smart contracts on the Stacks blockchain.",
+  "clojure_lsp": "https://github.com/clojure-lsp/clojure-lsp\nClojure Language Server",
+  "cmake": "https://github.com/regen100/cmake-language-server\nCMake LSP Implementation",
+  "cobol_ls": "Cobol language support",
+  "codebook": "https://github.com/blopker/codebook\n An unholy spell checker for code.\n`codebook-lsp` can be installed by following the instructions [here](https://github.com/blopker/codebook/blob/main/README.md#installation).\nThe default `cmd` assumes that the `codebook-lsp` binary can be found in `$PATH`.",
+  "coffeesense": "https://github.com/phil294/coffeesense\nCoffeeSense Language Server\n`coffeesense-language-server` can be installed via `npm`:\n```sh\nnpm install -g coffeesense-language-server\n```",
+  "contextive": "https://github.com/dev-cycles/contextive\nLanguage Server for Contextive.\nContextive allows you to define terms in a central file and provides auto-completion suggestions and hover panels for these terms wherever they're used.\nTo install the language server, you need to download the appropriate [GitHub release asset](https://github.com/dev-cycles/contextive/releases/) for your operating system and architecture.\nAfter the download unzip the Contextive.LanguageServer binary and copy the file into a folder that is included in your system's PATH.",
+  "copilot": "https://www.npmjs.com/package/@github/copilot-language-server\nThe Copilot Language Server enables any editor or IDE\nto integrate with GitHub Copilot via\n[the language server protocol](https://microsoft.github.io/language-server-protocol/).\n**[GitHub Copilot](https://github.com/features/copilot)**\nis an AI pair programmer tool that helps you write code faster and smarter.\n**Sign up for [GitHub Copilot Free](https://github.com/settings/copilot)!**\nPlease see [terms of use for GitHub Copilot](https://docs.github.com/en/site-policy/github-terms/github-terms-for-additional-products-and-features#github-copilot)",
```

Comment:
Nice, this would solve issue I noticed with https://github.com/nix-community/nixvim/pull/3743